src/main/java/ee/tuleva/onboarding/auth/mobileid/MobileIDSession.java

```diff
@@ -0,0 +1,53 @@
+package ee.tuleva.onboarding.auth.mobileid;
+
+import ee.sk.mid.MidHashToSign;
+import java.io.Serializable;
+import lombok.Getter;
+
+@Getter
+public class MobileIDSession implements Serializable {
+  private static final long serialVersionUID = -7501351267187058440L;
+
+  private String firstName;
+  private String lastName;
+  private String personalCode;
+  private String challenge;
+  private String sessionId;
+  private String phoneNumber;
+  private MidHashToSign authenticationHash;
```

Comment:
The `MidHashToSign` does not declare a `serialVersionUID` so session deserialization might break in the future (if we upgrade the `mid-rest-java-client` dependency and they've changed the class for example). Something to keep in mind (or to send a PR to https://github.com/SK-EID/mid-rest-java-client/blob/master/src/main/java/ee/sk/mid/MidHashToSign.java)

Reply:
Created a new issue
https://github.com/SK-EID/mid-rest-java-client/issues/20